tests/test_pep8.py

```diff
@@ -31,6 +31,7 @@
     '.git',
     'dependencies',
     'lib',
+    'src/opendr/perception/panoptic_segmentation/efficient_ps/algorithm/EfficientPS',
```

Comment:
In the ReadMe you mentioned that the code was modified to comply with pep8, so it shouldn't be skipped

Reply:
Sorry, this has to be removed from the readme. Forgot that I mentioned it there before adding this line as asked by Stefania.